test/distributed/_tensor/test_math_ops.py

```diff
@@ -187,6 +187,12 @@ def _replicate_fn(name, module, device_mesh):
                 device_mesh, placements=[Replicate()]
             )
 
+            from torch.distributed._tensor.placement_types import TensorMeta
+
+            dtensor_meta = y_dist._spec.tensor_meta
+            assert isinstance(dtensor_meta, TensorMeta)
+            # make sure the right shape in sharding prop
+            self.assertEqual(y_local.shape, dtensor_meta.shape)
```

Comment:
did you check if without this PR change this would fail?

Reply:
Yes unfortunately this check would fail without the change. The meta shape is 20,1,1 but the tensor shape is 20, 4, 8